lib/src/prepare_cronet.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Contains the nessesary setup code only. Not meant to be exposed.
+
+import 'dart:io' show Directory, File, Process, ProcessResult, ProcessStartMode;
+
+// import 'package:path/path.dart';
+
+import 'find_resource.dart';
+
+// TODO: Change URL and Version
+
+const _release = '1.0.0';
+const _cronetBinaryUrl =
+    'https://github.com/unsuitable001/dart_cronet_sample/releases/download/$_release/';
+final _cBinExtMap = {
+  'linux64': '.tar.xz',
+  'windows64': '.tar.gz',
+  'androidarm64-v8a': '.tar.xz',
+};
+
+const _cronetVersion = '"86.0.4240.198"';
+
+/// Builds the [wrapper] shared library
+/// according to [build.sh] file
+void buildWrapper() {

```

Comment:
Code that is never used by dependent libraries and is only used by `dart run cronet` should live inside the `bin/` folder rather than the `lib/` folder.